changelogs/fragments/5439-dig-return-empty-result.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - dig lookup plugin - add option to return empty result without empty strings, and return empty list instead of NXDOMAIN. (https://github.com/ansible-collections/community.general/pull/5439 https://github.com/ansible-collections/community.general/issues/5428)
```

Comment:
```suggestion
  - dig lookup plugin - add option to return empty result without empty strings, and return empty list instead of ``NXDOMAIN`` (https://github.com/ansible-collections/community.general/pull/5439, https://github.com/ansible-collections/community.general/issues/5428).
```

Reply:
changed